AFQ/api.py

```diff
@@ -48,6 +48,12 @@ def do_preprocessing():
 BUNDLES = ["ATR", "CGC", "CST", "IFO", "ILF", "SLF", "ARC", "UNC",
            "FA", "FP"]
 
+CALLOSUM_BUNDLES = ["AntFrontal", "Motor", "Occipital", "Orbital",
+                    "PostParietal", "SupFrontal", "SupParietal",
+                    "Temporal"]
+
+DEFAULT_BUNDLES = BUNDLES + CALLOSUM_BUNDLES
```

Comment:
This is a bit tricky. IIUC, this segmentation is not perfectly compatible with the default segmentation. 

That's because the CC bundles in the "standard"/default segmentation are partially overlapping with some of these custom CC bundles. So, if they are entered into segmentation together, they will "compete" for streamlines. 

So, I think that we'd want users to choose whether they want to segment CC *or* they want the standard segmentation.

Reply:
👍 Figured that might be the case.